service/storage_service.cc

```diff
@@ -1915,23 +1920,31 @@ future<std::unordered_set<token>> storage_service::prepare_replacement_info(cons
     // make magic happen
     slogger.info("Checking remote features with gossip");
     return _gossiper.do_shadow_round().then([this, loaded_peer_features, replace_address] {
-        auto local_features = get_known_features();
-        _gossiper.check_knows_remote_features(local_features, loaded_peer_features);
+        _gossiper.check_knows_remote_features(get_known_features(), loaded_peer_features);
+
```

Comment:
Why do you add this change? Even if you have a very good reason to do such refactoring, it should not be mixed in the function changes. Such changes makes the review much harder.

Reply:
I split the refactoring to a different commit.